src/_guides/language/analysis-options.md

```diff
@@ -376,11 +376,48 @@ linter:
 ```
 
 {{site.alert.note}}
-  Due to YAML restrictions, 
+  Due to YAML restrictions,
   **you can't mix list and key-value syntax in the same `rules` entry.**
-  You can, however, use the other syntax for rules in an included file.
+  You may use the other syntax for rules in an included file.
 {{site.alert.end}}
 
+## Enabling analyzer plugins (experimental) {#plugins}
+
+The analyzer has experimental support for plugins.
+These plugins integrate with the analyzer to add functionality
+such as new diagnostics, quick fixes, and custom code completion.
+You can enable only one plugin per `analysis_options.yaml` file.
+Enabling an analyzer plugin increases how much memory the analyzer uses.
+
+Don’t use analyzer plugins if your situation meets
+either of the following conditions:
+
+* You use a development machine with less than 16 GB of memory.
+* You use a mono-repo with more than 10 `pubspec.yaml` and
+  `analysis_options.yaml` files.
+  
+You can find a few analyzer plugins on
+[pub.dev]({{site.pub-pkg}}?q=dependency%3Aanalyzer_plugin).
+
+To enable a plugin:
+
+1. Add the package containing the plugin as a dev dependency.
```

Comment:
We haven't been super consistent about this, but with ordered lists, the first line of each entry should begin with one space. This prevents additional space being added to the beginning of the code snippets.

We document this at https://github.com/dart-lang/site-shared/blob/main/doc/markdown.md#do-properly-indent-markdown-lists

```suggestion
 1. Add the package containing the plugin as a dev dependency.
```